react/tic-tac-toe/src/app/screens/Game/components/TopBar/styles.module.scss

```diff
@@ -0,0 +1,13 @@
+@import '../../../../../scss/colors';
+
+.navbar {
+  box-shadow: 2px 2px 13px 1px $black-07;
+  height: 40px;
+  padding: 0;
+
+  .option {
```

Comment:
Avoid nesting classes. It increases specificity and makes troublesome to change them later. Be name specific instead.